t/30-mmapi.t

```diff
@@ -20,11 +20,50 @@
 
 use Test::Most;
 
+BEGIN {
+    $ENV{OS_AUTOINST_LOCKAPI_RETRY_COUNT}    = 1;
+    $ENV{OS_AUTOINST_LOCKAPI_RETRY_INTERVAL} = 0;
+}
+
 use OpenQA::Test::TimeLimit '5';
 use Test::Output;
 use Test::MockModule;
 use Mojolicious;
 use mmapi;
+use lockapi;
+use testapi;
+use autotest;
+use basetest;
```

Comment:
You don't need any of these imports if you just mock them. I tried it out, deleted all three, tests still pass :)